autorally_core/src/StateEstimator/BlockingQueue.h

```diff
@@ -90,6 +90,18 @@ class BlockingQueue {
     return boost::optional<WorkType>(element);
   }
 
+  // TODO this might not make sense - will assess later
```

Comment:
I was wrong, it is used.  This breaks the initial design goal of the queue class, but maybe that is ok in our case